Changelog.md

```diff
@@ -2,6 +2,11 @@
 
 # CassKop Cassandra Kubernetes Operator Changelog
 
+## 0.4.0
+
+- Uses of initContainerImage and bootstrapContainerImage to be able to adapt to official cassandra image.
+- Use of ReadOnly Container : `Spec.ReadOnlyRootFilesystem` default true
```

Comment:
- initContainerImage and bootstrapContainerImage used to ...
- ReadOnly Container